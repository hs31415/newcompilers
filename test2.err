Line: 4 Quote didn't match.Line: 8 '&&' didn't match.