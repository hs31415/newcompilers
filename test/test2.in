char c = 'AA';
for (integer i = 0; i < 10; i = i + 1) {
    write(c % 1);
}