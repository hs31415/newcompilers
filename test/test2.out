 <char>
<id, c>
 <=>
Line: 1 Single Quote didn't match.
 <for>
 <(>
 <integer>
<id, i>
 <=>
<NUM, 0>
 <;>
<id, i>
 <<>
<INT, 10>
 <;>
<id, i>
 <=>
<id, i>
 <+>
<INT, 1>
 <)>
 <{>
 <write>
 <(>
<id, c>
 <%>
<INT, 1>
 <)>
 <;>
 <}>
